### ISSUE TITLE
Atom Lite detection fails with Pro-DOS-v2

### ISSUE THREAD

**Issue by MrBlinky** [simonowen/simcoupe#65]
Pro-DOS-v2  doesn't detect a flash card after boot.
typing disk on pro-dos command line reports No flash cards

**Comment by simonowen** [comment]
Thanks for the report.  I can see some port errors reported when booting Pro-DOS-v2 in the latest version:
```
AtomLite: Unhandled write to d8f0 with d8
AtomLite: Unhandled write to d0f0 with d0
AtomLite: Unhandled write to 03f0 with 03
```

It works fine in a SimCoupe build version from last year so I've broken it somewhere along the way. I'll investigate...

**Comment by MrBlinky** [comment]
Thanks for looking into it so fast. Those are from the 2nd floppy drive test. Don't think they cause the issue.

**Comment by simonowen** [comment]
Ah yes, you're right! Thanks for saving me from investigating those unhandled writes :)

I've narrowed down the breaking change to 9b21a49a3f28d43ab30dc84149545cde2ee17dbd, which adds support for multi-sector reads. I think the problem is that when any data request has completed it's returning the next sector from the disk. This isn't right for IDENTIFY reads, and I think that leads to it failing. Most programs won't notice or care but I know Chris is quite thorough with his device status checks!

I'll double-check the modified behaviour with other applications tomorrow before committing the fix. I don't think I have any other changes pending but it's worth an updated version.

**Comment by MrBlinky** [comment]
Is that code is (re)used for ther identify command? Maybe  it's a missing m_sRegs.bSectorCount =1 on the identify command then?

**Comment by simonowen** [comment]
Yes, all buffer reads go through the same logic. Setting the sector count was a tempting solution, but the spec says identify doesn't use it so I wouldn't want to modify the current value.  Just suppressing the multi-sector behaviour for non-read commands seems to work.
